### PR TITLE
LPS-128322

### PR DESCRIPTION
**Pull request by erickmonteiroo**

### CODE REVIEW

**Comment by drewbrokke** [comment]
ci:test:sf

**Comment by drewbrokke** [comment]
ci:test:relevant

File: modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_users_admin/js/AccountUserEmailDomainValidator.es.js [hunk 1]
```diff
@@ -17,35 +17,46 @@ import {Config} from 'metal-state';
...
-						this.addFieldMessage_(field);
+		if (form) {
+			this.decorateEmailAddressField_(form);
+		}
+		else {
+			Liferay.once(
+				this.ns('formReady'),
+				(event) => {
+					const form = this.getForm_(event.formName);
+					if (form) {
```
Comment (drewbrokke) [inline modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_users_admin/js/AccountUserEmailDomainValidator.es.js:30]: SF: insert line break between 29 and 30

File: modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_users_admin/js/AccountUserEmailDomainValidator.es.js [hunk 2]
```diff
@@ -17,35 +17,46 @@ import {Config} from 'metal-state';
...
+	}
+
+	decorateEmailAddressField_(form) {
+		const field = form.formValidator.getField(this.ns('emailAddress'));
 
-					const emailDomainFieldRule = this.getEmailDomainFieldRule_();
+		if (this.viewValidDomainsURL) {
+			this.addFieldMessage_(field);
+		}
+		const emailDomainFieldRule = this.getEmailDomainFieldRule_();
```
Comment (drewbrokke) [inline modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_users_admin/js/AccountUserEmailDomainValidator.es.js:45]: SF: insert line break between 44 and 45

File: modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_users_admin/js/AccountUserEmailDomainValidator.es.js [hunk 3]
```diff
@@ -17,35 +17,46 @@ import {Config} from 'metal-state';
...
-			},
-			this
+		this.setWarningValidationStyle_(
+			form,
+			field,
+			emailDomainFieldRule.validatorName
 		);
 	}
 
+	getForm_(formName) {
```
Comment (drewbrokke) [inline modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_users_admin/js/AccountUserEmailDomainValidator.es.js:56]: SF: sort these methods with the others alphabetically:
- `getForm_`
- `decorateEmailAddressField_`

**Review by drewbrokke — CHANGES_REQUESTED** [review]
@erickmonteiroo just a few source formatting changes needed. Please see my inline comments.

**Comment by drewbrokke** [comment]
ci:forward

@erickmonteiroo looks good!

**Comment by Sharryxue** [comment]
ci:forward

**Comment by drewbrokke** [comment]
ci:forward

**Comment by Sharryxue** [comment]
ci:forward

**Comment by erickmonteiroo** [comment]
ci:test:relevant

**Comment by Sharryxue** [comment]
No related failure found, forward manually [brianchandotcom/liferay-portal#102557](https://github.com/brianchandotcom/liferay-portal/pull/102557)

**Comment by Sharryxue** [comment]
ci:close

**Pull request closed without merging**